assets/js/base/components/cart-checkout/address-form/address-form.tsx

```diff
@@ -281,17 +179,19 @@ const AddressForm = ( {
 										: newValue,
 							} )
 						}
+						customFormatter={ ( value: string ) => {
```

Comment:
Introduced a `customFormatter` callback so we can handle certain fields here, instead of handling it in the onChange function.